### DISCUSSION TITLE
Scripts in nocodb [May 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2022] [Discord Threads]
> 🙋‍♂️ **CaptainKranch**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Its not yet avaialble  
#  
> 💁 **o1lab**  
#### but you can use our APIs and webhooks  
#  
> 🙋‍♂️ **CaptainKranch**  
#### am I able to get information from an external api and put it on nocodb with the method get using webhooks?  
#  
> 🙋‍♂️ **CaptainKranch**  
#### ```json   
#### {"msg":"insert into \"nc_hook_logs_v2\" (\"base_id\", \"created_at\", \"event\", \"execution_time\", \"id\", \"notification\", \"operation\", \"payload\", \"project_id\", \"response\", \"test_call\", \"triggered_by\", \"type\", \"updated_at\") values ($1, CURRENT_TIMESTAMP, $2, $3, $4, $5, $6, $7, $8, DEFAULT, $9, $10, $11, CURRENT_TIMESTAMP) - invalid input syntax for type integer: \"460.735\""}   
#### ```  
#  
> 🙋‍♂️ **CaptainKranch**  
#### I'm getting this msg when I try to use the webhooks :/  
#  
> 💁 **o1lab**  
#### please paste your project info (right top menu)  
#  
> 🙋‍♂️ **CaptainKranch**  
#### Node: **v12.22.12**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **pg**   
#### ProjectOnRootDB: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.90.7**  
#  
> 🙋‍♂️ **BoMBx**  
#### According to this error message, it means you passing the wrong type of parameter. The input need interger type (...,-2,-1,0,1,2,...) but you pass the float (460.735). Or another way, just go to change type in your field of table on database into decimal type (or type which allow floating number).  
#  
> 🙋‍♂️ **BoMBx**  
#### <@177618286756757507>  
#  
> 🙋‍♂️ **CaptainKranch**  
#### I do not think thats the issue, I created a new table with no rows tried to use the webhook to give me a notification through email but it displays the same error... I saw that in github there are some similar errors should I open an issue on github?  
#  
> 💁 **o1lab**  
#### please do open an issue with the details  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968232693437464726)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
